### DISCUSSION TITLE
Update of dcm4che to Version 5

### DISCUSSION THREAD

**Discussion by lampi87** [nextgenhealthcare/connect#4699] [Feature/Enhancement Ideas]
Hello everyone,

As dcm4che2 is deprecated is there any plan to switch to dcm4che 5?
Or is it already supported?

Best regards,
Alex

**Comment by evulhotdog** [comment]
Would love to hear what the plan is, if any, being that dcm4che has been EOL for quite some time.

**Comment by cturczynskyj** [comment]
Converting this to a discussion as that is a better format for questions.

**Comment by evulhotdog** [comment]
With this now being a discussion, can NextGen speak to any DICOM related enhancements?

**Comment by 1-0-1developer** [comment]
I would really like to know this too, I have just started a project using the DICOM sender and receiver and came here after finding out version 2.x is used in Mirth instead of 5.x.

**Comment by fraser-nds** [comment]
Hi, would also be interested in seeing this happen. Can't use DICOM listener, because only supported version is TLS1.1 which is now deprecated.

**Comment by fraser-nds** [comment]
Hullo, any update on whether this could be added to a roadmap?
